war/src/main/webapp/css/style.css

```diff
@@ -1975,3 +1975,23 @@ body.main-panel-only #main-panel {
         padding: 2em 10%;
     }
 }
+
+img.icon-sm {
+    width: 16px;
+    height: 16px;
+}
+
+img.icon-md {
+    width: 24px;
+    height: 24px;
+}
+
+img.icon-lg {
+    width: 32px;
+    height: 32px;
+}
+
+img.icon-xl {
```

Comment:
I was mainly just restricting this to contain the impact of the change, but I'll modify as you suggest.